### ISSUE TITLE
No files found to analyze

### ISSUE THREAD

**Issue by karenpayneoregon** [ErikEJ/SqlServer.Rules#122]
While under C:\Users\paynek\Documents\SQL Server Management Studio, asked to analyze BirthDateGroupBy.sql. Also, several other files failed with one statement.

Error message

```
error: BirthDateGroupBy.sql is not a valid file path.
error: No files found to analyze
```

```sql
SELECT * FROM dbo.Birthdays
WHERE BirthDate IN
(
    SELECT BirthDate
    FROM dbo.Birthdays
    GROUP BY BirthDate
    HAVING COUNT(BirthDate) >= 0
)
ORDER BY dbo.Birthdays.BirthDate
```

The following failed with the same error message above. I could see perhaps as maybe an issue e.g. missing GO but the SQL runs fine in SSMS.

```sql
DECLARE @Ago int = -4;

SELECT Id, FORMAT(SomeDate, 'MM/dd/yyyy')
FROM dbo.WorkingWithDates1
WHERE SomeDate > DATEADD(day, @Ago, CAST(GETDATE() AS DATE))
ORDER BY SomeDate;

SELECT DATEADD(day, -4, CAST(GETDATE() AS DATE)) AS [Four days ago];
SELECT FORMAT(DATEADD(dd, DATEDIFF(dd,0,GETDATE()), 0), 'MM/dd/yyyy') AS [Start of week]
```

**Comment by ErikEJ** [comment]
@karenpayneoregon Please share your exact command line

**Comment by karenpayneoregon** [comment]
tsqlanalyze -i BirthDateGroupBy.sql

![Image](https://github.com/user-attachments/assets/bdfddbf4-8dbf-46cd-974d-2f97be843f24)

**Comment by karenpayneoregon** [comment]
Oh darn, was in the wrong folder, works in the right folder.
